lib/core/src/lib/translation/translate-loader.service.ts

```diff
@@ -103,7 +103,7 @@ export class TranslateLoaderService implements TranslateLoader {
     }
 
     isComponentInQueue(lang: string, name: string) {
-        return !!(this.queue[lang] || []).find((x) => x === name);
+        return !!(this.queue[lang] || []).some((x) => x === name);
```

Comment:
Method some already returns a boolean so there is no need for such conversion
```suggestion
        return (this.queue[lang] || []).some((x) => x === name);
```